scripts/shared/clusters.sh

```diff
@@ -304,13 +302,28 @@ function warn_inotify() {
     fi
 }
 
+# If any of the clusters use OVN-K as the CNI then clone the
+# ovn-kubernetes repo from master in order to access the required
+# kind scripts, and manifest generation templates.
+function download_ovnk() {
+    if [[ ${cluster_cni[*]} != *"ovn"* ]]
+    then
+        return
+    fi
+
+    echo "Cloning ovn-kubernetes from source"
+    rm -rf ovn-kubernetes/
```

Comment:
Should probably add `ovn-kubernetes` to `.gitignore`?

Reply:
Yep :)